.circleci/config.yml

```
@@ -87,14 +88,14 @@ jobs:
           command: docker network create potsie
       - run:
           name: Bootstrap project
-          command: make bootstrap
+          command: make bootstrap run-edx
       - run:
           name: Run e2e tests
-          command: make run-edx test
+          command: make test
```

To ensure edx is running, what about the following instead (along with removing the `run-edx` target in the previous step)?

```suggestion
          command: make run-edx && make test
```